aws/alarms/lambda/notify_slack/notify_slack.js

```diff
@@ -55,6 +55,64 @@ function getSNSMessageSeverity(message) {
   return "info";
 }
 
+function sendToOpsGenie(logGroup, logMessage, logLevel, context) {
+
+  const weShouldPageOnCall = (logLevel) => {
```

Comment:
The log level's current setting at 'error' may be too permissive. Hence, it is worth considering the addition of a 'SEV1' designation to the logs that necessitate immediate attention from the on-call team - TBD

Reply:
We already have a specific `severity` field in the SEV1 logs that you could parse and leverage.

Here is an example:

````
console.error(
      JSON.stringify({
        level: "error",
        severity: 1,
        msg: "Failed to run Audit Logs Processor.",
        error: (error as Error).message,
      })
);
`````